src/index.tsx

```diff
@@ -5,9 +5,30 @@ import App from './App'
 import SampleApp from './SampleApp'
 
 import * as serviceWorker from './serviceWorker'
+import {
+  AriaScreenReader,
+  SpeechSynthesisTextToSpeech,
+} from './utils/ScreenReader'
+import memoize from './utils/memoize'
+import { getUSEnglishVoice } from './utils/voices'
+
+const isSampleApp = window.location.hash === '#sample'
+const readerEnabled = window.location.search === '?reader=on'
```

Comment:
this is gonna bust if we ever add another parameter. How about using:
https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams

Reply:
I'm also wondering what happens if we ever reload the page for any reason (a fault, etc.) Don't have to solve it now, but maybe flag that for later.

Reply:
Good idea, done.